### ISSUE TITLE
Invalid default BatchSize for EventSourceMapping with SQS Queue

### ISSUE THREAD

**Issue by allison-knauss** [localstack/localstack#1720]
When attempting to locally start a stack containing an `AWS::Lambda::EventSourceMapping` resource that uses an SQS queue as its event source, the stack fails to start unless the optional `BatchSize` property is specified.

Logged error:
```
2019-11-04T19:56:14:DEBUG:localstack.services.cloudformation.cloudformation_starter: Currently updating stack resource minimal-local/InputQueueEventSourceMapping: None
2019-11-04T19:56:14:ERROR:localstack.services.cloudformation.cloudformation_starter: Unable to parse and create resource "InputQueueEventSourceMapping": ('InvalidParameterValueException', 'BatchSize 100 exceeds the max of 10') Traceback (most recent call last):
  File "/opt/code/localstack/localstack/services/cloudformation/cloudformation_starter.py", line 173, in parse_and_create_resource
    return _parse_and_create_resource(logical_id, resource_json, resources_map, region_name)
  File "/opt/code/localstack/localstack/services/cloudformation/cloudformation_starter.py", line 233, in _parse_and_create_resource
    resource_json_arns_fixed, resources_map, region_name)
  File "/opt/code/localstack/.venv/lib/python3.6/site-packages/moto/cloudformation/parsing.py", line 306, in parse_and_create_resource
    resource_name, resource_json, region_name)
  File "/opt/code/localstack/localstack/services/cloudformation/cloudformation_starter.py", line 472, in Mapping_create_from_cloudformation_json
    return Mapping_create_from_cloudformation_json_orig(resource_name, cloudformation_json, region_name)
  File "/opt/code/localstack/.venv/lib/python3.6/site-packages/moto/awslambda/models.py", line 492, in create_from_cloudformation_json
    return EventSourceMapping(spec)
  File "/opt/code/localstack/.venv/lib/python3.6/site-packages/moto/awslambda/models.py", line 452, in __init__
    "BatchSize {} exceeds the max of {}".format(batch_size, batch_size_entry[1]))
ValueError: ('InvalidParameterValueException', 'BatchSize 100 exceeds the max of 10')

```

This is the serverless.yml:

```
service: minimal

plugins:
  - serverless-localstack
  - serverless-pseudo-parameters

custom:
  localstack:
    host: http://localhost
    autostart: true
    lambda:
      mountCode: False
    stages:
      - local

provider:
  name: aws
  runtime: python3.7
  versionFunctions: false

  iamManagedPolicies:
    - arn:aws:iam::aws:policy/AWSLambdaExecute # Managed Policy

  iamRoleStatements:
    - Effect: Allow
      Action:
        - sqs:ReceiveMessage
        - sqs:DeleteMessage
        - sqs:GetQueueAttributes
      Resource: "#{InputQueue.Arn}"

functions:
  HelloFunction:
    handler: handler.hello
    name: "HelloFunction"

resources:
  Resources:

    InputQueue:
      Type: AWS::SQS::Queue
      Properties:
        QueueName: "InputQueue"

    InputQueueEventSourceMapping:
      Type: AWS::Lambda::EventSourceMapping
      DependsOn:
        - InputQueue
        - HelloFunctionLambdaFunction
      Properties:
        EventSourceArn: "#{InputQueue.Arn}"
        FunctionName: "#{HelloFunctionLambdaFunction}"
```

If I add `BatchSize: 10` after the last line, it works correctly.

This only occurs when deploying to the local stack. Deploying to AWS correclty sets the default when not specified.

**Comment by dtrts** [comment]
I'm finding this issue also. When creating a new event source mapping, despite providing a batch size, the new object always has a batch size of 100. It then has to be updated manually.

**Comment by mrosales** [comment]
This was seemingly "fixed" about a month ago, however it still does not match actual AWS behavior.

Expected behavior, [from the docs](https://docs.aws.amazon.com/lambda/latest/dg/API_UpdateEventSourceMapping.html) is:

* Amazon Kinesis - Default 100. Max 10,000.
* Amazon DynamoDB Streams - Default 100. Max 1,000.
* Amazon Simple Queue Service - Default 10. Max 10.

The change introduced in [a7f9ae2d](https://github.com/localstack/localstack/commit/a7f9ae2d2f1899d188fa2f4d36aea3160bcf69ca) dropped the default for all services to 10, which is at least a valid option for all, but I think this ticket should probably remain open until the behavior is consistent with AWS.

**Comment by whummer** [comment]
Thanks for reporting @allison-knauss @dtrts @mrosales . This should be fixed in #2260 - can you please give it a try with the `latest` version? Please report here if the problem persists. Thanks
